boa/src/context.rs

```diff
@@ -222,6 +222,39 @@ impl StandardObjects {
 /// `Context`s constructed in a thread share the same runtime, therefore it
 /// is possible to share objects from one context to another context, but they
 /// have to be in the same thread.
+///
+/// # Examples
+///
+/// ## Execute Function of Script File
+///
+/// ```rust
+/// use boa::{Context, exec::Executable, object::ObjectInitializer, property::Attribute};
+///
+/// let script = r#"
+/// function test(arg1) {
+///     if(arg1 != null) {
+///         return arg1.x;
+///     }
+///     return 112233;
+/// }
+/// "#;
+///
+/// let statements = boa::parse(script, false).unwrap();
+///
+/// let mut context = Context::new();
+/// // Populate the script definition to the context.
+/// statements.run(&mut context);
```

Comment:
```suggestion
/// use boa::{Context, object::ObjectInitializer, property::Attribute};
///
/// let script = r#"
/// function test(arg1) {
///     if(arg1 != null) {
///         return arg1.x;
///     }
///     return 112233;
/// }
/// "#;
///
/// // Populate the script definition to the context.
/// context.eval(script).unwrap();
```
we can call `eval` here too this will do the samething.

Reply:
Thanks for pointing out. I'll fix this.